test/gtest/ucp/test_ucp_am.cc

```diff
@@ -856,15 +862,87 @@ UCS_TEST_P(test_ucp_am_nbx, rx_am_mpools,
 UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx)
 
 
-class test_ucp_am_nbx_send_copy_header : public test_ucp_am_nbx {
+class test_ucp_am_nbx_single_multi_reply : public test_ucp_am_nbx {
 public:
+    test_ucp_am_nbx_single_multi_reply()
+    {
+        modify_config("RNDV_THRESH", "inf");
+        modify_config("PROTO_ENABLE", "y");
```

Comment:
need variants to test will run with both modes (y/n)

Reply:
You want me to fix this in this PR? This line of code is not part of the PR.